### DISCUSSION TITLE
Access without an optical connection?

### DISCUSSION THREAD

**Discussion by beriszb** [Anime4000/RTL960x#196] [Q&A]
Hello!

[DFP-34G-2C2] 
I received the device today.
The MikroTik router detects it.
But the intrface is not running.
(the optical cable is not plugged in)
I just wanted to open the Web interface.

Is it not possible?

```
[admin@MikroTik] > /interface ethernet monitor sfp1 once ;
                    name: sfp1
                  status: no-link
        auto-negotiation: disabled
      sfp-module-present: yes
             sfp-rx-loss: yes
                sfp-type: SFP-or-SFP+
      sfp-connector-type: SC
     sfp-link-length-9um: 20000m
         sfp-vendor-name: ODI
  sfp-vendor-part-number: DFP-34G-2C2
       sfp-vendor-serial: GPON23081195
  sfp-manufacturing-date: 23-08-19
          sfp-wavelength: 1310nm
         eeprom-checksum: good
                  eeprom: 0000: 03 04 01 00 00 00 02 22  00 01 00 01 0d 00 14 c8  ......." ........
                          0010: 00 00 00 00 4f 44 49 20  20 20 20 20 20 20 20 20  ....ODI          
                          0020: 20 20 20 20 00 00 00 00  44 46 50 2d 33 34 47 2d      .... DFP-34G-
                          0030: 32 43 32 20 20 20 20 20  20 20 20 20 05 1e 00 5f  2C2          ..._
                          0040: 00 1a 00 00 47 50 4f 4e  32 33 30 38 31 31 39 35  ....GPON 23081195
                          0050: 20 20 20 20 32 33 30 38  31 39 20 20 00 00 00 e2      2308 19  ....
                          0060: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          0070: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          0080: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          0090: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          00a0: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          00b0: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          00c0: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          00d0: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          00e0: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                          00f0: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
```

**Comment by vuducdong** [comment]
@beriszb Fiber optic cable needs to be plugged in (required)

**Comment by beriszb** [comment]
@vuducdong 
Thank you very much for your reply!
This is reassuring in that the device is not faulty.
But I am not currently where the optical cable ends.
@Anime4000 
Is there any way to reach the device until then?
What is the reason for this?

**Reply by Anime4000** [reply]
Mikrotik Frimware update before 7.8 no need fiber to be plug-in,
ROS after 7.8 require it, so far RTK SDK can't do fake RX signal

I guess ROS tries to make wide xPON compatibility and 2.5G support

only way is to own Media Converter

**Reply by beriszb** [reply]
Hi @Anime4000 !
I have installed ROS v7.7.
(previously ROS was v6.48.6!)

What can I change????

Router type: hEX S [RB760iGS].

This is the result:

```
  MMM      MMM       KKK                          TTTTTTTTTTT      KKK
  MMMM    MMMM       KKK                          TTTTTTTTTTT      KKK
  MMM MMMM MMM  III  KKK  KKK  RRRRRR     OOOOOO      TTT     III  KKK  KKK
  MMM  MM  MMM  III  KKKKK     RRR  RRR  OOO  OOO     TTT     III  KKKKK
  MMM      MMM  III  KKK KKK   RRRRRR    OOO  OOO     TTT     III  KKK KKK
  MMM      MMM  III  KKK  KKK  RRR  RRR   OOOOOO      TTT     III  KKK  KKK

  MikroTik RouterOS 7.7 (c) 1999-2023       https://www.mikrotik.com/

Press F1 for help



[admin@MikroTik] > /interface ethernet monitor sfp1 once ;
                      name: sfp1
                    status: no-link
          auto-negotiation: done
               advertising: 10M-half,10M-full,100M-half,100M-full,1000M-half,1000M-full
  link-partner-advertising: 
        sfp-module-present: yes
               sfp-rx-loss: yes
                  sfp-type: (unknown)
           eeprom-checksum: good
                    eeprom: 0000: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ........ ........
                            *
[admin@MikroTik] > 

```

**Comment by beriszb** [comment]
Another solution thread:

MC220L 
Gigabit SFP Media Converter
https://www.tp-link.com/en/business-networking/accessory/mc220l/

@Anime4000 
Would this solve the problem?
Would this allow me to access your web interface without an optical cable?

**Comment by vuducdong** [comment]
@beriszb

- RouterOS7 (below v7.8) only applies to specific models, for example: RB5009, but other models are still required to attach fiber optic cables.
- Similar to Media converters, not all models and different brands can be applied without attaching fiber optic cables.
- Some models tested ok (no need to attach fiber optic cable): Optone OPT-2200A, Juplink, Hellotek T8501S

**Reply by beriszb** [reply]
Thank you very much!
At the moment, I could only get a TP-Link [MC220L v4.20] Media Converter quickly.
It works for now!
I will report back with further developments.

**Comment by beriszb** [comment]
Hello!
@Anime4000 
@vuducdong 

We are making progress!
With the TP-Link [MC220L v4.20] Media Converter I was able to log in via the Web interface and SSH! :) 

I just need to configure the following parameters according to my provider:

 SN: 4857544354C000A3
MAC: C0:F6:EC:3A:71:2E

I think I can do that here: http://192.168.1.1/gpon.asp

But exactly at which points???
- "MAC" -vs- "MACKEY" ?
- "GPON SN:" -vs- "Device Serial Number" ?

Also, what does "Fiber Reset: Disable/Enable" mean?

![image](https://github.com/Anime4000/RTL960x/assets/145512206/eed48786-fad1-4732-aae9-4be19f21dad8)

**Reply by Anime4000** [reply]
MAC is the `Mac Address`
MAC KEY is the Firmware/Device License Key based on given `Mac Address` which is to prevent changing MAC Address on commercial/production

Fiber Reset not sure, if anyone here know what is that, tell us.
